plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -847,6 +844,10 @@ export class SessionRecordingIngester {
         )
     }
 
+    private isDebugLoggingEnabled(partition: number) {
+        return this.debugPartition === partition || this.config.SESSION_RECORDING_DEBUG_ALL
```

Comment:
Personally I prefer the idea of having the env var be `*` or `1` or `1,2,3` as that gives the most flexibility for the least config options...

Reply:
do you know... my inner ben said that too
he's easier to ignore 🙈 🤣